e2e/test/scenarios/dashboard-cards/dashboard-card-resizing.cy.spec.js

```diff
@@ -255,7 +255,7 @@ describe("scenarios > dashboard card resizing", () => {
 
   it(
     "should not allow cards to be resized smaller than min height",
-    { requestTimeout: 15000, tags: "@slow" },
```

Comment:
idk, if we put 15s of awaiting, can it be considered as not slow?

Reply:
🤷  slow test was slow, something had to go

Reply:
We had to put this back when GitHub CI was running on potato runners so the actual time to even fire a request was really long.
It's wise to leave it in there just in case we get a bad runner once in a while, but that doesn't mean that the test is always going to be slow.

I hope this makes sense.